src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.operate = (callback, number) => {
+    calculator.result = callback(calculator.result, number);
+
+    return calculator;
+  };
+
+  calculator.add = (a, b) => a + b;
+
+  calculator.subtract = (a, b) => a - b;
+
+  calculator.multiply = (a, b) => a * b;
+
+  calculator.divide = (a, b) => a / b;
```

Comment:
Fix it. 
<img width="815" alt="image" src="https://github.com/user-attachments/assets/a3673665-e5b0-4d0d-abf1-9be2d224aba5">